flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -162,6 +162,12 @@ class SentryFlutterOptions extends SentryOptions {
   /// Only available on Android.
   bool enableNdkScopeSync = false;
 
+  /// Automatically track app start measurement and send it with the
+  /// first transaction. Set to false when configuring option to disable or if
+  /// you want to set the end time of app startup manually using
+  /// [SentryFlutter.setAppStartEnd].
+  bool autoAppStart = true;
```

Comment:
on React Native this is called `enableAutoPerformanceTracking` and it's meant for both features, this one and Slow/Frozen frames, wdyt?

Reply:
If we combine the two, users won't be able to set values manually, so we won't enable the native side collection if `enableAutoPerformanceTracking` is false.

Reply:
Introduced this on the https://github.com/getsentry/sentry-dart/pull/772 PR.